### ISSUE TITLE
[NPC]Bladespire_Supplicant (BB #670)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#670]
This issue was migrated from bitbucket.
**Original Reporter:** silvermcd123
**Original Date:** 25.08.2014 01:50:38 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/670

<hr>

[NPC]Bladespire_Supplicant

Location: Circle of Blood, Blades Edge Mountains.
Problem: Should have appeared after quest Into the Soulgrinder, but it didnt appear.

http://www.wowhead.com/npc=23053

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** jeff_
**Original Date:** 02.09.2014 07:41:10 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/670/#comment-12109231

<hr>

found something at TC:
https://github.com/TrinityCore/TrinityCore/commit/4c69e04a0f30eafdb411cafbe3dd393b673f3ffe

Fix tested and I think it works... :)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 02.09.2014 09:46:40 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/670/#comment-12111846

<hr>

Sounds good. I have a lot of DB stuff I want to get going. Should fix a lot of issues and make the core better to use.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 08.10.2014 05:00:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/670/#comment-12838288

<hr>

[DB/Misc] Misc world database updates. See below for full fix list.

```
      Add flying flag for Neltharaku
      Remove loot from Voidwalker Minions
      Remove loot from Corehounds
      Fix spell positions and linked spells for Lunar Fest. Invitations
      Set sheath state for various creatures in Slave Pens
      Update boss waypoints in Slave Pens
      Update creature speeds in Arcatraz
      Correct mode for Felmist Gas Clouds
      Update OfferRewardText for quest 396
      Add spell dummy condition for spell Radiation
      Fix prevQuestID for Ogre Heaven. Fixes #670
      Update models and inhabit types for various triggers
      Correct InhabitType for Mana Snappers in Netherstorm
      Set correct speed for creatures in Blades Edge
      Fix Pink Elekks being visible when drunk
      Add missing areatrigger_tavern entries
      Remove weapon from Horde Orphans
      Update loot chance for item Orders from Kael'thas
      Remove Heavy Scorpid Vest from Portal Seekers and add to Shadowsworn Thug
      Fix NPCs involved in Garments of the Moon, Light AND Darkness
      Remove database spawned pets from Netherstorm, they're summoned by owners
      Fix a spawnmask for an NPC, who never would have spawned otherwise
      Add pathing to a Silvermoon City Guardian who was standing still
      Play intense battle music when starting the bombing quest on IoQD
      Fix various NextQuestIDs and PrevQuestIds
      Fix inhabit type for Floating Flavor Eyes
      Respawn Deadmire and add waypoints. Previously, he fell through the world
      Update creature addon template for Dormant Inferals
      Fix cosmetic work on Shadowmoon Harbingers
      Remove duplicate of Swirling Maelstrom in Ashzara
      Correct restock times of Darkmoon Faire vendor, Lhara
      Set ExclusiveGroup for Shattrath Daily quests
```

→ <<cset c5fc64ae6634>>
